pkg/cli/images_detail.go

```diff
@@ -66,8 +67,16 @@ func (a *ImageDetails) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	type ImageDetailsOutput struct {
+		internalv1.AppImage
+		SignatureDigest string `json:"signature,omitempty" yaml:"signature,omitempty"`
```

Comment:
If we do end up putting the `SignatureDigest` into the `AppImage` struct this change will likely become easier